.github/scripts/branch/checkBranch.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 
+acronym="[a-z]{3}"
+slash="\/"
+numbers="[0-9]+"
+hyphen="-"
+word="[^[:cntrl:]]+"
+
+regex="$acronym$slash$numbers$hyphen$word"
```

Comment:
I think that extracting this line to separate variable is redundant, it could be inlined.

Reply:
Done 😄